fluid-project/hearth
# feat: add ResourceCollection model and migration (resolves #134)

Adds ResourceCollection model with a pivot table for setting many to many relationship with Resource model. 

**Testing**
- Verify that translatable fields of ResourceCollection model is translatable
- Verify that resource collection belongs to a user get deleted when user is deleted
- Verify that resource collection can have many resources
- Verify that deleting a resource belongs to resource collection deletes their relationship
- Update test case for resource model to verify that single resource can be in many resource collections
- Update test case for resource model to verify that deleting a resource belongs to a resource collection deletes their relationship
